cmd/podman/pod_create.go

```diff
@@ -19,10 +19,11 @@ var (
 	DefaultKernelNamespaces = "cgroup,ipc,net,uts"
 )
 
-var podCreateDescription = "Creates a new empty pod. The pod ID is then" +
-	" printed to stdout. You can then start it at any time with the" +
-	" podman pod start <pod_id> command. The pod will be created with the" +
-	" initial state 'created'."
+var podCreateDescription = "Creates a new pod. The pod ID is then" +
+	" printed to stdout. If an infra container is attached to the pod (as is default)," +
+	" the pod will have an initial state of 'running', as the infra container is started after it's created." +
+    " Otherwise, the pod will have an initial state of 'created', and can be started with 'podman pod start <POD ID>'" +
+    " after containers are attached to it."
```

Comment:
I'd vote on holding this PR until @baude is able to get his cobra #2274 in

Reply:
But I want to make @baude 's life extra difficult :(

Reply:
I also believe the way this works is correct.  What happens if I reboot the machine and then add containers to an existing pod?

Reply:
Wouldn't work without a start on the infra container, at least.

Reply:
Right if I start a container in a pod and the pod is not running, the podman run or start should start the pod.

Reply:
I can work on a fix, but it'd be good to know if it's worth making a special case for pods (i.e: if a container is in a pod and the infra container isn't started, start the infra), or if it should be a generalized recursive start (if the container is created with any dependencies, then a podman run should start all dependencies too). WDYT @rhatdan @mheon

Reply:
I'd think more generalized recursive start is correct. I'm debating whether it needs to use the container graph code that `pod start` uses - it's probably more correct that way, but would be more work, and I don't think it's strictly necessary

Reply:
Go with what a user would expect.
If I start a container within a pod, I would expect the container and pod to start, which means that pod and all its children would start.

Note: The real think we need to figure out is what happens when a container stops.  1 of three things happens.

1.  Pod Stops
2.  Pod restarts the container
3. Nothing.